myuw/views/page.py

```diff
@@ -55,23 +54,8 @@ def page(request,
             return failure
     log_session(netid, request.session.session_key, request)
 
-    if _is_mobile(request):
-        # On mobile devices, all students get the current myuw.  Non-students
-        # are sent to the legacy site.
-        try:
-            if is_oldmyuw_mobile_user():
-                logger.info("mobile user %s, redirect to legacy!" % netid)
-                return redirect_to_legacy_site()
-        except Exception:
-            log_exception(logger,
-                          '%s is_oldmyuw_mobile_user' % netid,
-                          traceback.format_exc())
-            logger.info("%s, redirected to legacy!" % netid)
-            return redirect_to_legacy_site()
-
-    else:
-        if is_oldmyuw_user():
-            return redirect_to_legacy_site()
+    if is_oldmyuw_user():
```

Comment:
You may want to remove def _is_mobile(request): ...

Reply:
I believe this is used in some other places such as : https://github.com/uw-it-aca/myuw/blob/master/myuw/logger/session_log.py#L27

Reply:
is_mobile = request.is_mobile or request.is_tablet  
this doesn't look like depending on the _is_mobile function defined in page.py